flake.nix

```diff
@@ -45,6 +45,23 @@
         wasm-pack
         wasm-bindgen-cli
       ];
+
+      bin = {
+        bash = "${pkgs.bash}/bin/bash";
+        cargo = "${pkgs.cargo}/bin/cargo";
+        wasm-pack = "${pkgs.wasm-pack}/bin/wasm-pack";
+      };
+
+      test-all =
+        pkgs.writeScriptBin "test:all" ''
+          #!${bin.bash}
+          echo "⚙️  Running cargo test"
+          ${bin.cargo} test
+
+          printf %"$COLUMNS"s | tr " " "-"
+          echo "⚙️  Running headless Wasm tests"
+          ${bin.wasm-pack} test --headless --chrome deterministic-bloom-wasm
+        '';
```

Comment:
@zeeshanlakhani this just gets exported into the environment

<img width="698" alt="Screenshot 2022-12-19 at 20 44 16" src="https://user-images.githubusercontent.com/1052016/208585597-5b7c485f-8bd7-4b1a-b60c-d4862751f4c4.png">

Reply:
dope. want to add this to the rust-template flake for rust+wasm @expede? I'd typically do this in make (and we do this in ci), so I can see the benefit for nix peeps.

Reply:
Sure, makes sense!

We have a much more elaborate version in the Haskell repo, but I want to clean that up and just publish a package at some stage.

Reply:
Ah nice. It’d have rusty things in it, but that’d
Be a nice add